index.html

```diff
@@ -0,0 +1,139 @@
+  <link rel="stylesheet" href="./CSS/style.css">
+</head>
+<body>
+  <header class="header">
+    <div class="social_media">
+      <ul class="media-nav d-flex flex-center">
+        <li class="media_link"><a href="https://web.facebook.com/asia.magana"><i class="fa fa-facebook"></i></a></li>
+        <li class="media_link"><a href="https://twitter.com/asia_magana"><i class="fa fa-twitter"></i></a></li>
+        <li class="media_link"><a href="#">English</a></li>
+        <li class="media_link"><a href="#">My page</a></li>
+        <li class="media_link"><a href="#">Logout</a></li>
+      </ul>
+    </div>
+    <nav class="menu">
+      <div class="hamburger d-flex flex-center">
+        <img src="./images/icons/Union.png" class="show-menu-btn" alt="union">
+        <img src="./images/icons/Icon@2x.svg" class="closed-menu-btn" alt="">
+        <div class="logo">
+          <img src="./images/gender_equality_2.png" alt="logo">
+        </div>
+        <ul class="nav_menu d-flex">
+          <li class="menu_link active "><a href="#Home">Home</a></li>
+          <li class="menu_link"><a href="./about.html">About us</a></li>
+          <li class="menu_link"><a href="#Program">Mission</a></li>
+          <li class="menu_link"><a href="#Team">Team</a></li>
+          <li class="menu_link"><a href="#Partner">Partners</a></li>
+          <li class="menu_link active"><a href="#Home">CloudFlare</a></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+
+  <main>
+    <section id="Home" class="home_section">
+      <div class="home_section_title">
+          <h2>"Hello! Sharing world"</h2>
+          <h1>GLOBAL ISSUES: GENDER EQUALITY AND<br>WOMEN'S EMPOWERMENT</h1>
+      </div>
+      <article>
+        <h4>Gender equality is a human right, but our world faces a persistent gap in access to opportunities and decision-making power for women and men.</h4>
+      </article>
+      <div>
+        <div class="info">
+          <h3>FROM (MOND 8AM) ~ (SUN 17AM)</h3>
+          <p>@ Full-Cycle product Development</p>
+        </div>
+      </div>
+    </section>
+
+    <section id="Program" class="program_section">
+        <h2>Why Choose Us?</h2>
+        <ul class="program">
+            <li class="program_1 d-flex flex-center">
+              <div>
+                <img src="https://fireart.studio/wp-content/uploads/2019/12/ic-choose-1.svg" alt="agile">
+              </div>
+              <h3>Agile team </h3>
+              <p>An experienced team dedicated to only your project. There is no micromanagement. </p>
+            </li>
+            <li class="program_2 d-flex flex-center">
+              <div>
+                <img src="https://fireart.studio/wp-content/uploads/2019/12/ic-choose-2.svg" alt="focus">
+              </div>
+              <h3>Focus on JS</h3>
+              <p>You will deal with a team of seasoned professionals who have been programing together for many years and understand each other. </p>
```

Comment:
- [ ] As mentioned before, be careful with the sizes. In this text, after you hover over it for the first time, you get some scrolling bars which is not something we want here. Kindly work on this issue.